gnome-image-installer/gnome-image-installer.c

```diff
@@ -374,6 +375,11 @@ gis_page_get_type();
       gis_store_set_image_uuid (uuid);
     }
 
+  udisks_client = udisks_client_new_sync (NULL, &error);
+  if (udisks_client == NULL)
```

Comment:
Arguably not a new bug, because nothing was checking before whether UDisksClient was NULL or not... but if udisks_client_new_sync can return NULL, is it worth moving the creation into gis_store, so that every time we attempt to use the UDisks client, we will re-try connecting to the daemon?